open_humans/management/commands/suspend_users.py

```diff
@@ -47,4 +47,4 @@ def handle(self, *args, **options):
         for user in users_to_suspend:
             user.is_active = False
             user.save()
-            print('{} (ID: {}) is suspended.'.format(user.username, user.id))
+            print('{} (ID: {}) is suspended.'.format(user.username, user.id)
```

Comment:
I don't think management commands are particularly important, but this looks like it might be a typo?

Reply:
Clearly that does not have a test.  It was for sure a typo, and I'll be pushing a fix momentarily.